### PR TITLE
Upgrade Gatsby and MUI to latest version

### PR DESCRIPTION
**Pull request by tanisha-agarwal**
Fixes https://github.com/pixie-io/px.dev/issues/57.

Signed-off-by: Tanisha Agarwal <tanisha.agarwal@joshtechnologygroup.com>

### CODE REVIEW

**Comment by htroisi** [comment]
@tanisha-agarwal - the netlify logs show the following error:
```
11:22:12 AM: error gatsby@4.19.2: The engine "node" is incompatible with this module. Expected version ">=14.15.0". Got "12.18.0"
```
I believe @postescu ran into this same issue when updating our `blog.px.dev` repo and the solution was to add [this .nvmrc file](https://github.com/pixie-io/blog.px.dev/blob/main/.nvmrc). Could you update this PR to add the `.nvmrc` file?

**Comment by tanisha-agarwal** [comment]
@htroisi added .nvmrc file

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.13.1
```
Comment (htroisi) [inline .nvmrc:1]:
```suggestion
lts/*
```

**Comment by htroisi** [comment]
@tanisha-agarwal - FYI, you can reproduce the lint errors in the netlify logs by running `yarn lint`.

**Review by htroisi — APPROVED** [review]

**Review by NickLanam — APPROVED** [review]
Looks great to me, and clean too!

The Snyk warning is in a buried sub-dependency; there's a pending pull request to fix it there. We should keep an eye on that.

I noticed two preexisting issues (for future PRs, prod does this too already):
* The footer links need `target="_blank"` to open in new tabs.
* The "Be a Pixienaut" button has a mostly-covered click region, because a transparent part of the image to its left renders over top of it. The button needs to be on top.

**Review by aimichelle — APPROVED** [review]

**Pull request merged**